frontend/src/styles/design-tokens/themes/light/semantic.css

```
@@ -0,0 +1,131 @@
+  --bg-inverse: 255 255 255; /* rgba(255, 255, 255, 1) */
+
+  /* Border inverse with opacity variants */
+  --bdr-inverse-80: rgba(255, 255, 255, 0.8);
+  --bdr-inverse-60: rgba(255, 255, 255, 0.6);
+  --bdr-inverse-40: rgba(255, 255, 255, 0.4);
+  --bdr-inverse-20: rgba(255, 255, 255, 0.2);
+
+  /* Noise texture mask */
+  --noise-mask: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='100' height='100' viewBox='0 0 100 100'%3E%3Cfilter id='noise'%3E%3CfeTurbulence type='fractalNoise' baseFrequency='1.67' numOctaves='3' stitchTiles='stitch'/%3E%3CfeColorMatrix type='saturate' values='0'/%3E%3CfeComponentTransfer%3E%3CfeFuncA type='discrete' tableValues='0 0 0 0 0 0 0 0 0 0 0 0 0 0 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0'/%3E%3C/feComponentTransfer%3E%3C/filter%3E%3Crect width='100%25' height='100%25' filter='url(%23noise)' fill='white'/%3E%3C/svg%3E");
+  --noise-color: rgba(0, 0, 0, 0.2);
+  --noise-color: color-mix(in srgb, var(--bdr-inverse) 20%, transparent);
```

Biome (2.1.2)

[error] Duplicate custom properties can lead to unexpected behavior and may override previous declarations unintentionally.

--noise-color is already defined here.

Remove or rename the duplicate custom property to ensure consistent styling.

(lint/suspicious/noDuplicateCustomProperties)